src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -420,6 +444,34 @@ private static void close(final Closeable res) {
             this.output = out;
             this.level = lvl;
         }
+
+        /**
+         * Checks if line is part of a stack trace and should be appended.
+         * @param string String to check
+         * @return boolean result
```

Comment:
@dean-e-clark `result` should be changed to `Result` - there is no particular reason behind that, just the common style followed in this repo